Ginger/Ginger/WindowExplorer/Common/ControlActionsPage.xaml.cs

```diff
@@ -207,6 +183,41 @@ private void AddActionButton_Click(object sender, RoutedEventArgs e)
             AEP.ShowAsWindow();
         }
 
+        private Act SetActionDetails(Act act)
+        {           
+            act.Active = true;
+            act.AddNewReturnParams = true;
+            act.Value = ValueTextBox.Text;
+
+            ElementLocator EL = (ElementLocator)mLocators.CurrentItem;
+
+            if ((mActions.CurrentItem).GetType() == typeof(ActUIElement))
+            {
+                //Set UIElement action locator
+                ActUIElement actUI = (ActUIElement)act;
+                actUI.ElementLocateBy = EL.LocateBy;
+                actUI.ElementLocateValue = EL.LocateValue;         
+                
+                //Set action unique input values
+                if (mActInputValues!=null)
```

Comment:
why Set action unique input values been done only for ActUIElement action? do it for any action and if old mechinsem will be used so mActInputValues will be null or empty